portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,18 @@
-#content {
+body {
+    margin: 0px;
+    background-color: 
```

Comment:
`background-color` isn't set :) Let's remove it or set it to a color you like...